kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiDatabaseMetaData.java

```diff
@@ -106,7 +106,7 @@ public String getCatalogSeparator() throws SQLException {
   }
 
   public String getCatalogTerm() throws SQLException {
-    return "instance";
+    return "catalog";
```

Comment:
what's the reason of rename this term ?

Reply:
Currently, Kyuubi supports Spark, Flink, Trino, Hive engines, except to Hive, all other engines use term `catalog`.